### PR TITLE
[FIX] extract-msg: Error in version 0.27.6, define max version in requirements.txt

### PR DESCRIPTION
**Pull request by victoralmau**
Error in last version of extract-msg dependency 0.27.6 (https://pypi.org/project/extract-msg/0.27.6/) and 0.27.5.
Define max version in requirements.txt about it.

Please, @pedrobaeza  @joao-p-marques review it.

@Tecnativa TT26362

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # mail_inline_css
 premailer
-extract-msg
+extract-msg<=0.27.5
```
Comment (joao-p-marques) [inline requirements.txt]: I just noticed that the last pipeline that ran "green" was using v0.27.4: https://travis-ci.org/github/OCA/social/jobs/731915721#L481
```suggestion
extract-msg<=0.27.4
```
Sorry for the misunderstanding

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Fast-tracking this for solving the red build:

/ocabot merge patch

**Comment by pedrobaeza** [comment]
@victoralmau it seems to not be enough

**Comment by joao-p-marques** [comment]
It seems like it is still picking v0.26.6: https://travis-ci.org/github/OCA/social/jobs/737963894#L528

**Comment by pedrobaeza** [comment]
Maybe another library pushing that version of dependencies?

**Comment by victoralmau** [comment]
@pedrobaeza Work fine now, can you review it and merge?

**Comment by pedrobaeza** [comment]
I see, this is because it has `MQT_DEP="PIP"` in Travis, so we need to change all the setup thing...

/ocabot merge nobump

**Review by joao-p-marques — APPROVED** [review]

**Pull request merged**